css-four-column-layout/styles.css

```diff
@@ -0,0 +1,58 @@
+.container {
+  position: relative;
+  width: 100%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.center-align {
+  align-content: center;
+}
+
+.justify-evenly {
+  justify-content: space-evenly;
+}
+
+.text-align {
+  text-align: center;
+}
+
+.title {
+  justify-content: center;
+  text-align: center;
+}
+
+.title > h1 {
+  font-family: Barlow, sans-serif;
+  font-weight: 800;
+}
+
+.content {
+  width: 100%;
+  margin: auto;
+}
+
+.phone-content {
+  flex-direction: column;
+  text-align: center;
+  max-width: 20%;
```

Comment:
make sure that your column classes have an explicit width set on them. Max:width is a limit to size, not quite a set size.